<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="movieImgLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>512</width>
     <height>512</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTreeView" name="experimentsTreeView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>811</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
  </widget>
  <widget class="QListView" name="centrosomeListView">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>570</y>
     <width>51</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="nucleiListView">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>570</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="nucleiTitleLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>56</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nuclei</string>
   </property>
  </widget>
  <widget class="QLabel" name="centrosomesTitleLbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>550</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Centrosomes</string>
   </property>
  </widget>
  <widget class="QLabel" name="frameTitleLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>56</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame</string>
   </property>
  </widget>
  <widget class="QListView" name="centrosomeListView_A">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>570</y>
     <width>71</width>
     <height>111</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListView" name="centrosomeListView_B">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>700</y>
     <width>71</width>
     <height>111</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="centrosomesTitleLbl_A">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>550</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QLabel" name="centrosomesTitleLbl_B">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>680</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QSlider" name="frameHSlider">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="MplWidget" name="mplDistance" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
